res/__tc/clang/2_test.sh

```
@@ -0,0 +1,27 @@
+set -ex
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)